dune-project

```diff
@@ -5,6 +5,79 @@
 (formatting
  (enabled_for ocaml))
 
-; TODO
-;
-; (generate_opam_files true)
+(generate_opam_files true)
+
+(license LGPL-2.1)
+(maintainers "The Coq development team <coqdev@inria.fr>")
+(authors "The Coq development team, INRIA, CNRS, and contributors")
+; This generates bug-reports and dev-repo
+(source (github coq/coq))
+(homepage https://coq.inria.fr/)
+(documentation "https://coq.github.io/doc/")
+(version dev)
+
+; Note that we use coq.opam.template to have dune add the correct opam
+; prefix for configure
```

Comment:
This is not added to `coq-doc.opam` anymore. Is that an issue or not?

Reply:
I don't see the `coq-doc` build having to call configure anymore, but indeed we should test that `dune build -p coq-doc` works correctly. Still coq-doc is broken for install I think so we may need to tweak this more.

Reply:
I'm OK with merging this PR with a broken `coq-doc.opam` since it was already the case before.